.github/workflows/ci_workflow.yml

```diff
@@ -33,23 +33,19 @@ jobs:
 
   pytest:
     runs-on: ubuntu-latest
-    services:
-      postgres:
-        image: postgres
-        env:
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
     steps:
     - uses: actions/checkout@v3
+    - name: Fix key permissions
+      run: |
+        chmod 600 ssl/server.key
+        sudo chown 999:999 ssl/server.key
```

Comment:
uid/gid 999? Do you know why it's that group and user?

Reply:
I was getting permission issues, and when I checked the uid of the postgres user, 999:999 was displayed. Switched it to that and the permission issues cleared up.
![999-999](https://github.com/MeltanoLabs/target-postgres/assets/46581584/ff4c777b-71db-43c5-925c-2df86b68674b)